src/components/MarkdownProvider/components/Typography.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { getColor } from '@zendeskgarden/react-theming';
+import { XXL, XL, LG, MD } from '@zendeskgarden/react-typography';
+
+const headerSpacing = (p: ThemeProps<DefaultTheme>) => {
+  return css`
+    margin-top: ${p.theme.space.md};
+    margin-bottom: ${p.theme.space.sm};
+    padding-bottom: ${p.theme.space.xs};
+  `;
+};
+
+export const StyledH2 = styled(XXL).attrs({ tag: 'h2' })`
+  ${headerSpacing}
+`;
+
+export const StyledH3 = styled(XL).attrs({ tag: 'h3' })`
+  ${headerSpacing}
+`;
+
+export const StyledH4 = styled(LG).attrs({ tag: 'h4' })`
+  ${headerSpacing}
+`;
+
+export const StyledH5 = styled(LG).attrs({ tag: 'h5' })`
+  ${headerSpacing}
+`;
+
+export const StyledH6 = styled(LG).attrs({ tag: 'h6' })`
+  ${headerSpacing}
+`;
+
+export const StyledP = styled(MD).attrs({ tag: 'p' })`
+  /*
+   * May be removed once
+   * https://github.com/zendeskgarden/react-components/pull/724
+   * is released.
+   */
+  & + & {
+    margin-top: ${props => props.theme.lineHeights.md};
+  }
+`;
+
+export const StyledBlockquote = styled.blockquote`
+  margin-top: ${p => p.theme.space.md};
+  margin-bottom: ${p => p.theme.space.md};
+  border-left: ${p => p.theme.borders.sm} ${p => getColor('grey', 300, p.theme)};
+  padding-left: ${p => p.theme.space.sm};
+  color: ${p => getColor('grey', 600, p.theme)};
+  font-style: italic;
```

Comment:
It was mentioned in one of my PRs to avoid using italics in Garden due to i18n issues. Would that apply here too?

Reply:
This is specifically for the "emphasis" usage which is a valid markdown element. If we were to localize this content then the markdown source could not include that element if necessary.

Reply:
@hzhu the design decision was to use `italic` in this specific case. Use of italic on the product side continues to be a no-no :wink: